packages/core/src/internal-components/panel/panel.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+
+<Meta title="Internal/APIs/Panel/Getting Started" />
+
+<cds-alert-group status="danger" cds-layout="m-b:lg">
+  <cds-alert>This is an internal API and should only be used for developing Clarity Components.</cds-alert>
+</cds-alert-group>
+
+# Panel
+
+## Installation
+
+```typescript
+import '@cds/core/internal-components/panel/register.js';
+```
+
+## Usage
+
+```html
+<cds-internal-panel>
+  // content
+</cds-internal-panel>
+```
+
+<Preview>
+  <Story id="internal-apis-panel-stories--basic" />
+</Preview>
+
+## CSS Custom Properties
+
+- `--background`
+- `--color`
+- `--border-color`
+- `--border-width`
+- `--border-radius`
+- `--padding`
+- `--box-shadow`
+- `--overflow`
+- `--overflow-x`
+- `--overflow-y`
+- `--width`
+- `--height`
```

Comment:
You don't have to list these out, instead you can generate a API table with storybook

```tsx

## API

<Props of={'cds-internal-panel'} />
```